Homework11/LinksTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+
+namespace Homework11
+{
+    public class LinksTests
+    {
+        private IWebDriver _driver;
+        private IJavaScriptExecutor _javascriptExecutor;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            _driver = new ChromeDriver();
+            _javascriptExecutor = (IJavaScriptExecutor)_driver;
+            _driver.Manage().Window.Maximize();
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/elements");
+            _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            var navigationButton = _driver.FindElement(By.Id("item-5"));
+            ScrollToElement(navigationButton);
+            navigationButton.Click();
```

Comment:
You already included click action in ScrollToElement method. Please check where it's more appropriate to leave click action

Reply:
Fixed.